src/test/java/step1/SetTest.java

```diff
@@ -0,0 +1,51 @@
+package step1;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+
+    @Test
+    @DisplayName("Set의 size() 메소드를 활용해 Set의 크기를 확인한다.")
+    void size() {
+        int actual = numbers.size();
+        int expected = 3;
+
+        assertThat(actual).isEqualTo(expected);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3})
+    @DisplayName("Set의 contains() 메소드를 활용해 1, 2, 3의 값이 존재하는지를 확인한다.")
+    void contains1(int input) {
+        assertThat(numbers.contains(input)).isTrue();
```

Comment:
```suggestion
        assertThat(numbers).contains(input);
```

collection의 경우 위와 같이 구현도 가능함